Create recipe form template

shawnrhoads opened the pull request:
Added form templates for the following:
- blank issue
- bug issue
- recipe suggestion

shawnrhoads committed:
Create blank_issue.md

shawnrhoads committed:
Create issue-template.yml

shawnrhoads committed:
Create recipe-template.yml

dmoracze commented on .github/ISSUE_TEMPLATE/recipe-template.yml:
```diff
@@ -0,0 +1,47 @@
+    id: Language
+    attributes:
+      label: Language
+      description: What language are you using?
```
Might want to constrain tags to fit into neat categories rather than free text.

shawnrhoads committed:
added specific tags for the recipe submission template

added specific tags for language, software, and modality, but could not figure out how to allow user-specified tags

dmoracze reviewed (approved):
LGTM!
